__tests__/app.test.js

```diff
@@ -45,3 +45,72 @@ describe("GET /api/categories", () => {
 			});
 	});
 });
+
+describe("GET /api/reviews", () => {
+	test("status:200, responds with a reviews array", () => {
+		return request(app)
+			.get("/api/reviews")
+			.expect(200)
+			.then(({ body }) => {
+				const { reviews } = body;
+				expect(reviews).toBeInstanceOf(Array);
+				expect(reviews).toHaveLength(13);
+				reviews.forEach((review) => {
+					expect(review).toEqual(
+						expect.objectContaining({
+							owner: expect.any(String),
+							title: expect.any(String),
+							review_id: expect.any(Number),
+							category: expect.any(String),
+							review_img_url: expect.any(String),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							comment_count: expect.any(String)
+						})
+					);
+				});
+			});
+	});
+});
+
+describe("GET /api/reviews/:review_id", () => {
+	test("status:200, responds with a review object", () => {
+		return request(app)
+			.get("/api/reviews/2")
+			.expect(200)
+			.then(({ body }) => {
+				const { review } = body;
+				expect(review).toEqual({

```

Comment:
Ideally we would want to use objectContaining or toMatchObject in case in the future we wanted to add more properties 🙂